src/cysignals/macros.h

```diff
@@ -276,10 +304,21 @@ static inline void sig_error(void)
     {
         fprintf(stderr, "sig_error() without sig_on()\n");
     }
+#if defined(__MINGW32__) || defined(_WIN32)
+    /*
+     * The Windows abort function will terminate the process no
```

Comment:
Does the `abort()` function terminate the process or does the `SIGABRT` signal abort the process? If only `abort()` really aborts, then you could use `raise(SIGABRT)` instead.